### DISCUSSION TITLE
Disable the package version and other minor tweaks

### DISCUSSION THREAD

**Discussion by ghost** [starship/starship#3461] [Q&A] [answered]
How can I get the prompt to only show as the following

[the current directory open, in full without anything being cut off]
┌───────────────────>
└─>[where I enter the command I wish to use]

I don't wish to have extra things like package version, or git repository indicators, special character, etc.
If you don't mind helping me out, *Im kinda new to a lot of customization and stuff and I really love the starship prompt with what I got to work already... even if I just copied a preset of the website, still very a nice prompt*

**Comment by chipbuster** [comment, marked as answer]
Honestly, if all you want is the directory and the separator string, you might be better off using your shell's native prompt mechanism. Starship shines when there's a lot of complex environmental information to parse and display--using it for a simple text display seems slightly overkill.

If you really want to do this in starship, there are two things you'll want to look at:

1. Set `format` to something like the following (you might need to tweak it, I haven't tested this):

```
$directory
┌───────────────────>
└─>
```

2. Set `truncate_to_repo = false` to disable path truncation.

**Reply by ghost** [reply]
unfortunately the "truncate_to_repo = false" threw me an error but i took a look at what i could do with just the bash prompt and i got it to work, though i believe the font is differnet so it looks a little bit off and less nice, but works. Thank you for your time

**Reply by chipbuster** [reply]
@Grey-Gimple No worries!

Starship and bash themselves can't control the fonts, but it's possible that starship is setting bold/color control codes that make the font look slightly different. If you want, you can look at e.g. [this document](https://linuxhint.com/bash_test_background_colors/) to try to set color/bold the way you like it.
